dev-build/conf/application-logger.xml

```diff
@@ -11,7 +11,7 @@
         </rollingPolicy>
```

Comment:
While you are in there, if a developer needs a rolling policy on dev-build then they are in much more serious need of getting a life :wink: